src/report-portal.js

```diff
@@ -128,32 +128,48 @@ class ReportPortal {
     async _startFixturePreTest(time, name = "Before Test") {
         //need to close former fixture
         await this._finishFixture(time);
-
+        let fixtureDescription;
+        let hasAttribute = false;
+        if(name !== "Before Test"){
+    
+          fixtureDescription = `
+                ${name.split("\n").map(attr =>{
+                  return `* ${attr} \n`;
+                })}
+              `.replace(/\n,/g,"\n");
+          name = name.split("\n")[0];
+          hasAttribute = true;
+        }
+        
         if (this.launch !== undefined && this.launch.id !== undefined) {
-            const options = {
-                launchUuid: this.launch.id,
-                name: name,
-                startTime: time,
-                type: "before_test",
+          let options;
+          if (hasAttribute){
+            options = {
+              launchUuid: this.launch.id,
+              name: name,
+              startTime: time,
```

Comment:
can we write the payload once and only edit and add description? could be more elegant